content/guides/hot-module-replacement.md

```diff
@@ -213,7 +213,7 @@ __webpack.config.js__
       new webpack.HotModuleReplacementPlugin()
     ],
     output: {
-      filename: 'bundle.js',
+      filename: '[name].build.js',
```

Comment:
You are right it should be `[name].bundle.js` that is what's on the other guides, will update